x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectAuthenticator.java

```diff
@@ -514,12 +514,12 @@ private void handleTokenResponse(HttpResponse httpResponse, ActionListener<Tuple
                 return;
             }
             final Charset encoding = encodingHeader == null ? StandardCharsets.UTF_8 : Charsets.toCharset(encodingHeader.getValue());
-            final String json = EntityUtils.toString(entity, encoding);
             if (LOGGER.isTraceEnabled()) {
-                LOGGER.trace("Received Token Response from OP with status [{}] and content [{}] ",
-                    httpResponse.getStatusLine().getStatusCode(), json);
+                LOGGER.trace("Received Token Response from OP with status [{}]",
```

Comment:
We should print the content when the response status is `400` as it will contain the error : https://openid.net/specs/openid-connect-core-1_0.html#TokenErrorResponse . 

We should also do this on `WARN` level